extension/src/plots/model/custom.ts

```diff
@@ -130,26 +111,10 @@ export const createSpec = (title: string, metric: string, param: string) =>
           ]
         },
         mark: {
-          type: 'rule'
-        },
-        params: [
-          {
-            name: 'hover',
-            select: {
-              clear: 'mouseout',
-              fields: ['param', 'metric'],
-              nearest: true,
-              on: 'mouseover',
-              type: 'point'
-            }
-          }
-        ]
-      }
-    ],
-    transform: [
-      {
-        as: 'y',
-        calculate: "format(datum['y'],'.5f')"
+          filled: true,
+          size: 60,
+          type: 'point'
```

Comment:
Turns out just filling `point` or using `circle` make the marks even smaller:

<img width="653" alt="Screenshot 2023-04-05 at 9 09 11 AM" src="https://user-images.githubusercontent.com/43496356/230113372-d39630b4-6ebc-4409-91a4-ea8df7bb42b7.png">

I ended up increasing the size so that the filled points would have a similar width to none filled points:

<img width="683" alt="Screenshot 2023-04-05 at 9 13 40 AM" src="https://user-images.githubusercontent.com/43496356/230113511-d13e9298-daa3-4af9-a2b7-6469d8e5b482.png">

<img width="657" alt="Screenshot 2023-04-05 at 9 12 08 AM" src="https://user-images.githubusercontent.com/43496356/230113589-57c66495-29a2-46b3-89ef-c83ab51ba9c1.png">